tests/lockfile/Lockfile.test.ts

```diff
@@ -104,4 +96,43 @@ describe('Lockfile is', () => {
 
     await lockfile.stop();
   });
+  test('A running status holds a lock', async () => {
```

Comment:
This file should be changed to `tests/status/Status.test.ts`.

Also, given that the `Status` implementation was somewhat based off the refactored `Session` in the Gitlab MR, I believe there was some discussion from @CMCDragonkai about bringing over any relevant tests from [`Session.test.ts`](https://gitlab.com/MatrixAI/Engineering/Polykey/js-polykey/-/blob/26307a1ca449939198fd22c221e4b763b7597520/tests/sessions/Session.test.ts). Are there any specific extras you'd like to see here?

Reply:
I've fixed the test file name.
for the tests when I looked at the session tests I didn't see anything that wasn't already covered.